### ISSUE TITLE
Setting `debug: true` throws exception

### ISSUE THREAD

**Issue by kevinfarrugia** [getsentry/sentry-webpack-plugin#92]
Whenever adding the optional option `debug:true` to the SentryWebpackPlugin, it throws the following exception
```
TypeError: Cannot read property '0' of undefined
    at input.loaders.sillyClone.map.x (/Users/user/git/Sample/node_modules/@sentry/webpack-plugin/src/index.js:256:35)
    at Array.map (<anonymous>)
    at SentryCliPlugin.injectReleaseWithDebug (/Users/user/git/Sample/node_modules/@sentry/webpack-plugin/src/index.js:256:9)
    at SentryCliPlugin.apply (/Users/user/git/Sample/node_modules/@sentry/webpack-plugin/src/index.js:307:12)
    at Compiler.apply (/Users/user/git/Sample/node_modules/tapable/lib/Tapable.js:375:16)
    at webpack (/Users/user/git/Sample/node_modules/webpack/lib/webpack.js:33:19)
    at processOptions (/Users/user/git/Sample/node_modules/webpack/bin/webpack.js:329:15)
    at yargs.parse (/Users/user/git/Sample/node_modules/webpack/bin/webpack.js:387:2)
    at Object.Yargs.self.parse (/Users/user/git/Sample/node_modules/yargs/yargs.js:533:18)
    at Object.<anonymous> (/Users/user/git/Sample/node_modules/webpack/bin/webpack.js:152:7)
    at Module._compile (module.js:660:30)
    at Object.Module._extensions..js (module.js:671:10)
    at Module.load (module.js:573:32)
    at tryModuleLoad (module.js:513:12)
    at Function.Module._load (module.js:505:3)
    at Function.Module.runMain (module.js:701:10)
    at startup (bootstrap_node.js:190:16)
    at bootstrap_node.js:662:3
```

Extract from `webpack.config.js`:
```
                    new SentryWebpackPlugin({
                        release: 'test-1',
                        debug: true,
                        ignoreFile: '.sentrycliignore',
                        include: '.'
                    }),
```

**Comment by Ancinek** [comment]
Having the same issue :(

**Comment by rishantagarwal** [comment]
Downgrading to v5 solves it

**Comment by kamilogorek** [comment]
Fixed in `1.6.2`
